app/facades/food_facade.rb

```diff
@@ -0,0 +1,18 @@
+class FoodFacade
+
+  def self.top_foods(ingredient)
+    results = {hits: 0, foods: []}
+    data = FoodService.food_search_endpoint(ingredient)
+    results[:hits] = data[:totalHits]
+    data[:foods].each do |food|
+      food_data = {
+        gtinUpc: food[:gtinUpc],
+        description: food[:description],
+        brandOwner: food[:brandOwner],
+        ingredients: food[:ingredients]
+      }
+      results[:foods] << Food.new(food_data)
+    end
+    results
+  end
```

Comment:
Awesome 🎉  I really like that you've returned all data points for the user story with a single call to the API.